warehouse/templates/pages/help.html

```diff
@@ -86,8 +85,6 @@
 {% macro collaborator_roles() %}{% trans %}What collaborator roles are available for a project on PyPI?{% endtrans %}{% endmacro %}
 {% macro request_ownership() %}{% trans %}How do I become an owner/maintainer of a project on PyPI?{% endtrans %}{% endmacro %}
 {% macro description_content_type() %}{% trans %}How can I upload a project description in a different format?{% endtrans %}{% endmacro %}
-{% macro file_size_limit() %}{% trans %}How do I get a file size limit exemption or increase for my project?{% endtrans %}{% endmacro %}
-{% macro project_size_limit() %}{% trans %}How do I get a total project size limit exemption or increase for my project?{% endtrans %}{% endmacro %}
```

Comment:
Given https://github.com/pypi/warehouse/pull/17716/files/dfc8837225f0224d8e94b60e72b951e4075619fc..5d1452e32b44508906281208c9dcabc012c764ae#r1985565493, I would argue that we should add these back in a separate PR.

Reply:
I went ahead and re-added all three of these, with slightly tweaked language/links to the new user docs 🙂